README.md

```diff
@@ -56,9 +60,36 @@ gulp.task('sass:watch', function () {
 });
 ```
 
+You can choose whether to use [Dart Sass][] or [Node Sass][] by setting the `sass.compiler` property. Node Sass will be used by default, but it's strongly recommended that you set it explicitly for forwards-compatibility in case the default ever changes.
```

Comment:
The explicit compiler would be a breaking change and probably dealt with then instead of recommending it now. I'm just thinking if the future change happens it might be through a different method and this locks in the API when it's not needed yet

Reply:
One way or another, there needs to be a way of overriding the compiler. That's the whole point of this change.

Reply:
Technically overriding the compiler is already possible. Who knows what folks are doing with it. I'm OK with this because now we're at least documenting it as part of the API so we're committed to version correctly if we change.